src/main/java/store/reduct/client/ReductClient.java

```diff
@@ -56,6 +52,16 @@ public <T> HttpResponse<T> send(HttpRequest.Builder builder, HttpResponse.BodyHa
 		}
 	}
 
+	public <T> HttpResponse<T> send2(HttpRequest.Builder builder, HttpResponse.BodyHandler<T> bodyHandler) {
```

Comment:
send2 doesn't help to understand the difference with send1.  Give them please some sensible names and little documentation.

Reply:
Done